documentation/source/users/cantherm/input.rst

```diff
@@ -611,6 +611,14 @@ This is also acceptable::
 
     kinetics('H + C2H4 <=> C2H5')
 
+If a sensitivity analysis is desired, simply add the conditions at which to calculate sensitivity coefficients
```

Comment:
it might be useful to add the syntax for sensitivity using pressureDependence method (in addition to the kinetics method used here). 

we might also want to describe what the output of sensitivity analysis looks like in the documentation (including the folder name, how the sensitivity coefficients are normalized, etc.)